src/FileService.Mappers/Models/ContentTypeMapper.cs

```diff
@@ -0,0 +1,31 @@
+﻿using LT.DigitalOffice.FileService.Mappers.Models.Interfaces;
+using Microsoft.AspNetCore.StaticFiles;
+using MimeTypes;
+
+namespace LT.DigitalOffice.FileService.Mappers.Models
+{
+  public class ContentTypeMapper : IContentTypeMapper
+  {
+    private readonly FileExtensionContentTypeProvider _fileExtensionContentTypeProvider;
+
+    public ContentTypeMapper()
+    {
+      _fileExtensionContentTypeProvider =  new FileExtensionContentTypeProvider();
+    }
+
+    public string Map(string extension)
+    {
+      if (extension is null)

```

Comment:
can extension be empty? don't we need check in this case?

Reply:
Yes